### PR TITLE
Create base style package

### LINKED ISSUE #494: Create a dotcom-core-branding package
Page Kit aims to surface its contents to the top level such that the list of packages can be read as a table of contents for the project. 

To meet this goal, our core-branding, including our choices of fonts and colours, should be defined in a separate package and imported by the packages which require them.

### PR DESCRIPTION
**Pull request by i-like-robots**
This new package provides the global document styles for FT.com, including basic typography, normalisation styles, and grid.

The intention of this package is to centralise all of the required pieces for o-typography's progressive font loading and create a placeholder for the future inclusion of styles from `n-ui-foundations`.

It provides 3 parts, firstly the server-side integration:

```jsx
import {
  documentStyles
  LoadFontsEmbed,
  loadCustomFontsClassNames
} from '@financial-times/dotcom-ui-base-styles'

export default (props) => (
  <html className={loadCustomFontsClassNames} style={documentStyles}>
    <head>
      <meta charSet="utf-8" />
      <title>My Amazing Website</title>
      <LoadFontsEmbed />
    </head>
    <body>
      ...
    </body>
  </html>
)
```

Secondly, client-side JS:

```js
import * as baseStyles from '@financial-times/dotcom-ui-base-styles'

baseStyles.init()
```

And lastly, the styles:

```scss
@import '@financial-times/dotcom-ui-base-styles/styles';
```

Resolves #494

### CODE REVIEW

File: packages/dotcom-ui-base-styles/src/lib/documentStyles.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { CSSProperties } from 'react'
+
+export const documentStyles: CSSProperties = {
+  // Enable use of 100vw which does not account for the scroll bar
+  overflowX: 'hidden',
+  // o-colors "paper"
+  backgroundColor: '#fff1e5',
+  // o-colors "black-80"
+  color: '#33302e'
+}
```
Comment (andygout) [inline packages/dotcom-ui-base-styles/src/lib/documentStyles.ts]: What is going on in this file?

Comment (i-like-robots) [reply packages/dotcom-ui-base-styles/src/lib/documentStyles.ts]: These are the styles to attach to the document element... inlining them isn't strictly necessary but it does enable us to paint a pink background right away.

Comment (i-like-robots) [reply packages/dotcom-ui-base-styles/src/lib/documentStyles.ts]: I have updated (and corrected) the documentation to cover this.

File: packages/dotcom-ui-shell/src/components/Shell.tsx [hunk 2]
```diff
@@ -36,25 +42,15 @@ function Shell(props: TShellProps) {
...
       lang="en-GB"
-      className="no-js core"
-      style={{
-        // TODO: abstract styles into 'core branding' package
-        // Enable use of 100vw which does not account for the scroll bar
-        overflowX: 'hidden',
-        backgroundColor: '#fff1e5',
-        color: '#33302e'
-      }}>
+      className={`no-js core ${loadCustomFontsClassNames}`}
```
Comment (pixelandpage) [inline packages/dotcom-ui-shell/src/components/Shell.tsx]: Having access to the `<html>` element (and not relying on `.n-layout`) is so much nicer!

File: packages/dotcom-ui-base-styles/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "focus-visible": "^5.0.0"
+  },
+  "peerDependencies": {
+    "react": "^16.8.6"
+  },
+  "engines": {
```
Comment (i-like-robots) [inline packages/dotcom-ui-base-styles/package.json]: Needs React as a dev dependency

**Pull request merged**